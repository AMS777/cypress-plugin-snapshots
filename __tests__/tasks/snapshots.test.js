/* eslint no-template-curly-in-string: 0 */
describe('tasks/snapshots', () => {
  describe('applyReplace', () => {
    const {
      applyReplace
    } = require('../../tasks/snapshots');

    it('object replace', () => {
      const expected = {
        "text": "Hi ${name} & ${name}",
        "text2": "Hi ${name}",
      };
      const result = applyReplace(expected, {
        name: 'Meinaart',
      });
      expect(result).toMatchSnapshot();
    });
  });
})
