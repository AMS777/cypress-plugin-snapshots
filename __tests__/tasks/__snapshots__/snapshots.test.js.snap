// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tasks/snapshots applyReplace object replace 1`] = `
Object {
  "text": "Hi Meinaart & Meinaart",
  "text2": "Hi Meinaart",
}
`;
